src/main/java/de/uhd/ifi/se/decision/management/jira/consistency/implementation/TextualSimilarityCIP.java

```diff
@@ -0,0 +1,61 @@
+package de.uhd.ifi.se.decision.management.jira.consistency.implementation;
+
+import com.atlassian.jira.issue.Issue;
+import de.uhd.ifi.se.decision.management.jira.classification.preprocessing.Preprocessor;
+import de.uhd.ifi.se.decision.management.jira.consistency.ContextInformationProvider;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class TextualSimilarityCIP implements ContextInformationProvider {
+	private String id = "TextualSimilarityCIP_jaccard";
+	private String name = "TextualSimilarityCIP";
+
+	private static Preprocessor pp;
+
+	public TextualSimilarityCIP() {
+		if(pp == null){
+			pp  = new Preprocessor();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible unsafe assignment to a non-final static field in a constructor.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5455522)